sormas-backend/src/main/java/de/symeda/sormas/backend/symptoms/SymptomsFacadeEjb.java

```diff
@@ -37,7 +37,9 @@ public Symptoms fromDto(SymptomsDto source, boolean checkChangeDate) {
 			return null;
 		}
 
-		Symptoms target = DtoHelper.fillOrBuildEntity(source, service.getByUuid(source.getUuid()), Symptoms::new, checkChangeDate);
+		String uuid = source.getUuid();
```

Comment:
I'd prefer a more compact solution here because declaring a variable and then immediately reassigning looks a bit weird to me: 
```
String uuid = source.getUuid();
Symptoms target = DtoHelper.fillOrBuildEntity(source, uuid != null ? service.getByUuid(uuid) : null, Symptoms::new, checkChangeDate);
```